### ISSUE TITLE
Migrating SQLite to PosgreSQL failed due to table_shows.alternativeTitle not existing in PostgreSQL database.

### ISSUE THREAD

**Issue by captmicr0** [morpheus65535/bazarr#2436]
**Describe the bug**
When following the steps in the Wiki to migrate to PostgreSQL, pgloader fails with this error:
```
KABOOM!
SIMPLE-ERROR: pgloader failed to find column "public"."table_shows"."\"alternativeTitle\"" in target table "\"public\".\"table_shows\""
An unhandled error condition has been signalled:
   pgloader failed to find column "public"."table_shows"."\"alternativeTitle\"" in target table "\"public\".\"table_shows\""
```

**To Reproduce**
1. Follow steps in Wiki to migrate database to PostgreSQL
2. Try to complete the last step
```
docker run --rm -v /absolute/path/to/bazarr.db:/bazarr.db --network=host ghcr.io/roxedus/pgloader --with "quote identifiers" --with "data only" --cast "column table_blacklist.timestamp to timestamp" --cast "column table_blacklist_movie.timestamp to timestamp" --cast "column table_history.timestamp to timestamp" --cast "column table_history_movie.timestamp to timestamp" /bazarr.db "postgresql://_<postgres_username>_:_<postgres_password>_@hostname:5432/bazarr"
```

**Expected behavior**
The database to migrate properly.

**Software (please complete the following information):**
 - Bazarr: v1.4.2
 - OS: Docker container on Ubuntu 22.04

**Workaround**
To work around this error, I created the alternativeTitle column in the table_shows table, and then ran pgloader again.
It completed without error.
Upon inspecting the alternativeTitle column in the table, no data is there.
It seems like this column from the SQLite database MUST exist in the PostgreSQL database to migrate, but bazarr does not create it while creating the other tables on the first run when using PostgreSQL. No data actually inserted into it.
I deleted the column after migration, and bazarr has been running with no issues.

**Comment by morpheus65535** [comment]
You did this on an old bazarr.db? Because we used this column (`alternativeTitle`) a long time ago and it was suppose to be removed and replaced by `alternativeTitles`.

**Comment by captmicr0** [comment]
That is most likely the case, as the bazar.db I was migrating was originally created with a version about 1 year old.
I also might have skipped a few version when upgrading before migrating to Postgres.

**Comment by morpheus65535** [comment]
I'll see if I get another report of this issue. Until then, I'll close this issue.

**Comment by joszz** [comment]
Got exactly the same error while trying to do the exact same thing (move from sqlite to postgres). My original sqlite DB was created; 2022-11-21.
I´ĺl try to rename the table in the sqlite DB and see what happens

**Comment by captmicr0** [comment]
[joszz](https://github.com/joszz), if you want to re-try migration following these steps (exact steps I followed) to see if they work for you, that'd be great.
I did get a couple warnings from pgloader, but it didn't cause any issues. You can mostly ignore them I think.

When importing, I followed the steps laid out [here](https://www.reddit.com/r/sonarr/comments/16x0dlv/more_reliable_steps_for_migrating_your_sqlite/).

I modified them for bazarr, along with the temporary alternativeTitle column, below:

(for the the following, if you are using the postgres docker, you can docker exec -it into a shell and run them)
1.    Create the databases, e.g.
       `psql -U postgres_username -c "CREATE DATABASE bazarr;"`

2.    Configure the app to use postgres and start it, which will create the tables. Stop the app afterwards.

3.    Dump the schema:
       `pg_dump -U postgres_username bazarr -s > bazarr.sql`

4.    Drop the database and recreate it, e.g.
       `psql -U postgres_username -d bazarr -c "DROP DATABASE bazarr; CREATE DATABASE bazarr;"`

5.    Import the schema you've just dumped:
       `psql -U postgres_username bazarr < bazarr.sql`

6.    Add a column alternativeTitle in the "table_shows" table:
       `psql -U postgres_username -d bazarr -c "ALTER TABLE table_shows ADD alternativeTitle text;"`
(if using the postgres docker, you can exit the shell now)

7.    Import your old database using pgloader (see below).
```
docker run --rm \
  -v /path_to_database/bazarr.db:/bazarr.db:ro \
  --network=host ghcr.io/roxedus/pgloader \
  --with "prefetch rows = 100" \
  --with "batch size = 1MB" \
  --with "quote identifiers" \
  --with "data only" /bazarr.db \
  "postgresql://postgres_username:postgres_password@postgres_host:postgres_port/bazarr"
```

(again, if you are using the postgres docker, you can docker exec -it into a shell and run this)
8.  Remove the alternativeTitle in the "table_shows" table:
       `psql -U postgres_username -d bazarr -c "ALTER TABLE table_shows DROP COLUMN alternativeTitle;"`
(if using the postgres docker, you can exit the shell now)

**Comment by joszz** [comment]
Thx. that seems to work better!

                table name     errors       rows      bytes      total time
--------------------------  ---------  ---------  ---------  --------------
         Total import time          ✓      15408    28.7 MB          1.173s

**Comment by morpheus65535** [comment]
The problem is that most of those issue were fixed in database migration script but only if you were running postgresql. If you were using sqlite when database got upgraded, those fixes were not applied.

What happens if you empty the version number inside sqlite database before using pgloader?

`UPDATE alembic_version SET version_num = '';`

**Comment by morpheus65535** [comment]
This one should have been fixed at the same time as #2489 .
